jekyll/_cci2/concepts.md

```diff
@@ -89,54 +89,89 @@ Jobs are the building blocks of your config. Jobs are collections of [steps](#st
 ![job illustration]( {{ site.baseurl }}/assets/img/docs/job.png)
 
 #### Executors and Images
-{:.no_toc}
 
 Each separate job defined within your config will run in a unique executor. An executor can be a docker container or a virtual machine running Linux, Windows, or MacOS. Note, macOS is not currently available on self-hosted installations of CircleCI Server.
 
 ![job illustration]( {{ site.baseurl }}/assets/img/docs/executor_types.png)
 
 You can define an image for each executor. An image is a packaged system that has the instructions for creating a running container or virtual machine. CircleCI provide a range of images for use with the Docker executor. For more information see the [Pre-Built CircleCI Docker Images]({{ site.baseurl }}/2.0/circleci-images/) guide.
 
- ```yaml
- version: 2
- jobs:
-   build1: # job name
-     docker: # Specifies the primary container image,
-       - image: buildpack-deps:trusty
-
-       - image: postgres:9.4.1 # Specifies the database image
-        # for the secondary or service container run in a common
-        # network where ports exposed on the primary container are
-        # available on localhost.
-         environment: # Specifies the POSTGRES_USER authentication
-          # environment variable, see circleci.com/docs/2.0/env-vars/
-          # for instructions about using environment variables.
-           POSTGRES_USER: root
+{:.tab.executors.Cloud}
+```yaml
+version: 2.1
+
+jobs:
+ build1: # job name
+   docker: # Specifies the primary container image,
+     - image: buildpack-deps:trusty
+     - image: postgres:9.4.1 # Specifies the database image
+      # for the secondary or service container run in a common
+      # network where ports exposed on the primary container are
+      # available on localhost.
+       environment: # Specifies the POSTGRES_USER authentication
+        # environment variable, see circleci.com/docs/2.0/env-vars/
+        # for instructions about using environment variables.
+         POSTGRES_USER: root
 #...
-   build2:
-     machine: # Specifies a machine image that uses
-     # an Ubuntu version 14.04 image with Docker 17.06.1-ce
-     # and docker-compose 1.14.0, follow CircleCI Discuss Announcements
-     # for new image releases.
-       image: ubuntu-1604:201903-01
+ build2:
+   machine: # Specifies a machine image that uses
+   # an Ubuntu version 14.04 image with Docker 17.06.1-ce
+   # and docker-compose 1.14.0, follow CircleCI Discuss Announcements
+   # for new image releases.
+     image: ubuntu-1604:201903-01
 #...       
-   build3:
-     macos: # Specifies a macOS virtual machine with Xcode version 11.3
-       xcode: "11.3.0"
+ build3:
+   macos: # Specifies a macOS virtual machine with Xcode version 11.3
+     xcode: "11.3.0"
 # ...          
- ```
+```
+
+{:.tab.executors.Server}
+```yaml
+version: 2.0
```

Comment:
```suggestion
version: 2.1
```

Reply:
Ah yes this is tabbed content, and this is the server tab, so needs to stay at 2.0